types/istanbul-lib-report/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for istanbul-lib-report 1.1
+// Type definitions for istanbul-lib-report 3.0
```

Comment:
Just a remark: if I take sample from the project v3 readme:
https://github.com/istanbuljs/istanbuljs/blob/master/packages/istanbul-lib-report/README.md#example-usage
It does not compile due to missing props from the sample code: `defaultSummarizer`, `skipEmpty`, `skipFull`.

Reply:
Thank you, that's a good catch. 
I should have been a bit more diligent instead of just making the changes *I* wanted. This obviously needs a little more work.

Reply:
I should probably note that `skipEmpty` and `skipFull` aren't valid options for a `json` report